examples/gallery/index.html

```diff
@@ -36,11 +47,48 @@ <h1>Slint Gallery</h1>
     </a>
   </p>
   <script type="module">
-    import init from './pkg/gallery.js';
-    init().finally(() => {
-      document.getElementById("spinner").remove();
-    });
+    import initFluent from './pkg/fluent/gallery.js';
+    import initFluentLight from './pkg/fluent-light/gallery.js';
+    import initFluentDark from './pkg/fluent-dark/gallery.js';
+    import initMaterial from './pkg/material/gallery.js';
+    import initMaterialLight from './pkg/material-light/gallery.js';
+    import initMaterialDark from './pkg/material-dark/gallery.js';
```

Comment:
would there be a way to only load the alternative files when the combobox is changed, maybe?

Reply:
Yes, I think that would be possible using [dynamic imports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import). Those tend to be tricky with bundlers, but we're not bundling here - so it should be fine to use.